src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
+
+  &__images {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: $font-size-m;
+    line-height: $line-h-l;
+    margin: 0 0 4px;
+    transition: color $time-for-animation;
+  }
+
+  &:hover &__name {
+    color: #34568b;
```

Comment:
It's good practice to keep all styles related to the same element together. The `&:hover &__name` block should be moved directly after the `&__name` block to improve readability and maintainability.